packages/core/helper-plugin/lib/src/index.js

```diff
@@ -2,7 +2,7 @@ import { getType, getOtherInfos } from './content-manager/utils/getAttributeInfo
 
 // Contexts
 export { default as AppInfosContext } from './contexts/AppInfosContext';
-export { default as AutoReloadOverlayBockerContext } from './contexts/AutoReloadOverlayBockerContext';
+export { default as AutoReloadOverlayBlockerContext } from './contexts/AutoReloadOverlayBlockerContext';
```

Comment:
Both sides had the typo

Reply:
Can you still support the old naming so we avoid breaking changes? Can you add a comment as well?